msal/src/androidTest/java/com/microsoft/identity/client/DiscoveryTests.java

```diff
@@ -57,10 +58,8 @@ public void setUp() {
     }
 
     @Test
-    public void testInstanceDiscovery() throws IOException {
+    public void testInstanceDiscovery() throws IOException, URISyntaxException {
```

Comment:
suggestion: Tests can happily claim to throw Exception.  You'll never need to change them again.